vagrant/ansible/roles/client.test/tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Fetch the latest samba-integration-tests repo
+  git:
+    repo: 'https://github.com/spuiuk/samba-integration.git'
```

Comment:
We should not merge with using this git repo.
We now have the tests branch. And once there's something there that we can run as test, we can put the final repo here before merging.

Reply:
we can now use `https://github.com/gluster/samba-integration.git` as PR #50 is merged.

Reply:
Yes. The PR was labelled as do not merge for this reason. I put it up so Anoop and Gunther could test it.